### PR TITLE
Clarify inputs for functions

### PR DESCRIPTION
**Pull request by claucece**

### CODE REVIEW

File: draft-irtf-cfrg-voprf.md [hunk 1]
```diff
@@ -542,6 +542,9 @@ private and public key pair (skX, pkX), where skX is a random, non-zero
 element in the scalar field `GG` and pkX is the product of skX and the
 group's fixed generator.
 
+For unverifiable modes, servers don't need to make the resulting public key `pkS`
+accessible for clients.
```
Comment (armfazh) [inline draft-irtf-cfrg-voprf.md:546]:
```suggestion
For base mode, servers don't need the public key `pkS` produced by KeyGen.
```

**Review by armfazh — APPROVED** [review]

File: draft-irtf-cfrg-voprf.md [hunk 2]
```diff
@@ -542,6 +542,8 @@ private and public key pair (skX, pkX), where skX is a random, non-zero
 element in the scalar field `GG` and pkX is the product of skX and the
 group's fixed generator.
 
+For base mode, servers don't need the public key `pkS` produced by KeyGen.
```
Comment (chris-wood) [inline draft-irtf-cfrg-voprf.md:545]:
```suggestion
For base mode, servers do not need the public key `pkS` produced by KeyGen.
```

File: draft-irtf-cfrg-voprf.md [hunk 3]
```diff
@@ -987,11 +985,11 @@ def Unblind(pkS, token, blindToken, Ev):
 
 # Ciphersuites {#ciphersuites}
 
-A ciphersuite for the protocol wraps the functionality required for the
-protocol to take place. This ciphersuite should be available to both the
-client and server, and agreement on the specific instantiation is
-assumed throughout. A ciphersuite contains instantiations of the
-following functionalities.
+A ciphersuite (also referred as 'suite' on this document) for the protocol
```
Comment (chris-wood) [inline draft-irtf-cfrg-voprf.md:988]:
```suggestion
A ciphersuite (also referred as 'suite' in this document) for the protocol
```

**Review by chris-wood — APPROVED** [review]
LGTM pending the suggestions!

File: draft-irtf-cfrg-voprf.md [hunk 4]
```diff
@@ -987,11 +985,11 @@ def Unblind(pkS, token, blindToken, Ev):
 
 # Ciphersuites {#ciphersuites}
 
-A ciphersuite for the protocol wraps the functionality required for the
-protocol to take place. This ciphersuite should be available to both the
-client and server, and agreement on the specific instantiation is
-assumed throughout. A ciphersuite contains instantiations of the
-following functionalities.
+A ciphersuite (also referred as 'suite' in this document) for the protocol
```
Comment (chris-wood) [inline draft-irtf-cfrg-voprf.md:988]:
```suggestion
A ciphersuite (also referred to as 'suite' in this document) for the protocol
```

**Pull request merged**